### ISSUE TITLE
error

### ISSUE THREAD

**Issue by atilaro** [DnAp/Agar.io-Reverse-Engineering#26]
If you cannot connect to the servers, check if you have some anti virus or firewall blocking the connection.

**Comment by legos91** [comment]
Yes, I have such error to. I check if I'm connected to the game by https protocol I get an error. But when I change https to http there no error :)
